wApi/dot-net-api-setup.md

```diff
@@ -1,212 +1,267 @@
+Finally click **Apply** and then **OK** twice to save the changes.
 
-<!--Sql Server Authentication to MSSqlServer-->
-<add name="DataApi_UsrPwd" connectionString="Server=MyServerName; Database=MyDbName; User ID=MyUserName; Password=MySecret;" providerName="System.Data.SqlClient" />
+### Firewall
 
-<!--Using the MySQL connector plugin   (note providerName="MySql.Data.MySqlClient") see http://dev.mysql.com/downloads/connector/net/--> 
-<add name="DataApi_MySql"  connectionString="server=MyServerName; database=MyDbName; User Id=MyUserName; password=MySecret;" providerName="MySql.Data.MySqlClient" />
-    
-<!--ODBC driver to Oracle or MySQL-->
-<add name="DataApi_ODBC"   connectionString="Driver={Oracle in Ora_32bit}; dbq=MyDbName; Uid=MyUserName; Pwd=MySecret;"      providerName="System.Data.SqlClient" />
-```
+When setting up a website in Internet Information Services (IIS), you may need to edit your firewall permissions to allow incoming traffic to the web server. This is an important step in making your website accessible to users over the network. 
+
+### Testing
+
+At this point, the server in IIS should be running.
+
+![](/images/dot-net-api/IISServerRunning.png)
+<br>
+
+To confirm the API is working, navigate to the host name or port you set up [previously](#setup-website-in-iis) and access the status endpoint ("api/v1/status"). You should see "true" if the API is working.
 
+![](/images/dot-net-api/ApiRunning.png)
+<br>
+
+### Connection Strings
+
+The most common use of a Data API is for Intranet use. In this scenario a connection string can be shared by all users, while also protected within the Data API.
+
+Interject supports DataPortal connections for both direct DB access and API access. When using an API, you can code the API to perform certain DB actions by using "PassThrough" settings. These are configured in the DataPortal and DataPortal Connection. For these to work, they need to be able to find a connection string in the API.
+
+If this API offers public access, then connection strings should not be shared in the `web.config` file. Instead a custom validation lookup process should be implemented, which also requires some custom coding. This Data API uses a `appsettings.json` file for this purpose.
 
-### **13.  Additional settings in the web.config**
-In the web.config file there are app.settings which affect the behavior of the API.
-Double check that the web.config file has settings which match the ones below:
+* Determine a meaningful name for the connection string (preferably without spaces) This name will be used by an Interject DataPortal Connection, and it will reference the connection string within the `appsettings.json`. Names must be unique.
+* Open the `appsettings.json` file (in the `interject-dotnet-api` directory) in any text editor or code IDE.
+* The contents are formatted as XML. Locate the section <ConnectionStrings>.
+* Add a new line which uses the Name of the Connection String and its value.
```

Comment:
Probably should say: "Add an entry in the 'ConnectionStrings' collection." or something like that.